### PR TITLE
PIM-6897: As Julia, I would like to update the family variant labels from the UI

### PR DESCRIPTION
**Pull request by mmetayer**
This PR adds a translations tab in the 'Edit family variant' form

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | OK
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | OK
| Review and 2 GTM                  | OK
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: tests/legacy/features/family/edit_family_variant.feature [hunk 1]
```diff
@@ -48,3 +48,19 @@ Feature: Edit family variant
     Then the attribute "Brand" should be on the attributes level 2
     When I move the "Variation Name" attribute from level 1 to level 2
     Then the attribute "Variation Name" should be on the attributes level 2
+
+  Scenario: Successfully update family variant label translations
+    Given I am on the "Clothing" family page
+    And I visit the "Variants" tab
+    When I click on the "Clothing by color and size" row
```
Comment (phaseinducer) [inline tests/legacy/features/family/edit_family_variant.feature]: Still an "and". You should have only one "when" by scenario.

File: tests/legacy/features/family/edit_family_variant.feature [hunk 2]
```diff
@@ -48,3 +48,19 @@ Feature: Edit family variant
...
+    And I follow the link "Label translations"
+    When I fill in the following information in the popin:
+      | German (Germany)        | German label  |
+      | English (United States) | English label |
+      | French (France)         | French label  |
+    When I press the "Save" button in the popin
```
Comment (phaseinducer) [inline tests/legacy/features/family/edit_family_variant.feature]: And

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/family-variant/attribute-set.html [hunk 3]
```diff
@@ -1,57 +1,60 @@
...
-                    <span
-                        class="AknIconButton AknIconButton--small AknIconButton--delete AknButtonList-item delete-attribute-group"
-                        title="<%- __('pim_enrich.entity.family.variant.attribute_group_remove') %>"
-                    ></span>
-                    <% } %>
-                </span>
-                <%= renderSection(level.level, section.attributes, movable) %>
+        <div class="family-variant-level AknFamilyVariant-column AknFamilyVariant-column--level<%- level.level %>" data-level="<%- level.level %>">
+            <div class="AknFamilyVariant-axes AknVerticalList-item">
+                <%- level.axes.reduce(function (result, axis) {
```
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/family-variant/attribute-set.html]: here, you should use a map method, not a reduce :)

Comment (mmetayer) [reply src/Pim/Bundle/EnrichBundle/Resources/public/templates/family-variant/attribute-set.html]: Actually I only added the 2 first lines and the last one in this file, the remaining diff is only about indentation :)

Comment (mmetayer) [reply src/Pim/Bundle/EnrichBundle/Resources/public/templates/family-variant/attribute-set.html]: Done anyway

**Pull request merged**